File: PyDodo/pydodo/reset_simulation.py
import requests

from .config_param import config_param
from .utils import construct_endpoint_url

endpoint = config_param('endpoint_reset_simulation')

def reset_simulation():
    """
    Reset the simulation.
    """
    url = construct_endpoint_url(endpoint)
    resp = requests.post(url)

    # if response is 4XX or 5XX, raise exception
    resp.raise_for_status()
    return True

File: rdodo/R/reset_simulation.R
#' Reset the simulation
#'
#' @return
#' \code{TRUE} if successful. Otherwise \code{FALSE} and an error is thrown.
#'
#' @examples
#' \dontrun{
#' reset_simulation()
#' }
#'
#' @export
reset_simulation <- function() {

  post_call(endpoint = config_param("endpoint_reset_simulation"))
}
